module/Application/src/Service/Email.php

```diff
@@ -0,0 +1,121 @@
+    ) {
+    }
+
+    public function sendEmailTemplate(
+        MailAddress $recipient,
+        string $titleHeader,
+        string $titleBlock,
+        string $bodyMain,
+        ?string $titleAccessible = null,
+        ?string $titleMoreInformation = null,
+        ?string $bodyMoreInformation = null,
+        ?string $footerReason = null,
+        ?string $emailSubject = null,
+    ): void {
+        $replyTo = new MailAddress($this->config['to']['public']['address'], $this->config['to']['public']['name']);
+
+        $body = $this->render(
+            'email/basic',
+            [
+                'title_header' => $titleHeader,
+                'title_block' => $titleBlock,
+                'body_main' => $bodyMain,
+                'title_accessible' => $titleAccessible ?? $titleBlock,
+                'title_moreinformation' => $titleMoreInformation,
+                'body_moreinformation' => $bodyMoreInformation,
+                'footer_reason' => $footerReason,
+                'footer_sender_email' => $replyTo->getEmail(),
+            ],
+        );
+
+        $this->sendEmail(
+            $body,
+            $emailSubject ?? $titleHeader,
+            $recipient,
+            $replyTo,
+        );
+    }
+
+    private function sendEmail(
+        string $body,
+        string $subject,
+        MailAddress $recipient,
+        ?MailAddress $replyTo = null,
+    ): void {
+        $html = new MimePart($body);
+        $html->type = 'text/html';
+
+        $mimeMessage = new MimeMessage();
+        $mimeMessage->addPart($html);
+
+        $message = new Message();
+        $message->getHeaders()->addHeader((new MessageId())->setId());
+        $message->setBody($mimeMessage);
+        $message->setFrom($this->config['from']['address'], $this->config['from']['name']);
+        $message->setTo($recipient);
+        $message->setBcc($this->config['from']['address'], $this->config['from']['name']);
+        $message->setSubject($subject);
+
+        if (null !== $replyTo) {
+            $message->setReplyTo($replyTo);
+        }
+
+        $this->getMailTransport()->send($message);
+    }
+
+    /**
+     * Render a template with given variables.
+     *
+     * @param array<array-key,mixed> $vars
+     */
+    private function render(
+        string $template,
+        array $vars,
+    ): string {
+        $model = new ViewModel($vars);
+        $model->setTemplate($template);
+
+        return $this->renderer->render($model);
+    }
+
+    private function getMailTransport(): TransportInterface
+    {
+        return $this->mailTransport;
+    }
+
+    /**
+     * Get the storage config, as used by this service.
+     *
+     * @return array containing the config for the module
+     *
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingTraversableTypeHintSpecification
+     */
+    private function getConfig(): array
+    {
+        return $this->config['email'];
+    }
```

Comment:
It appears this function is never used, so can be removed.

Reply:
Done, https://github.com/GEWIS/gewisdb/pull/286/commits/76f4efbe7a8985a19a98271a70db39ac3aef9c75